torchdata/dataloader2/communication/iter.py

```diff
@@ -123,7 +123,12 @@ def DataPipeBehindQueues(source_datapipe, protocol, full_stop=False, blocking_re
             yield True
             continue
 
-        if isinstance(request, communication.messages.ResetIteratorRequest):
+        if isinstance(request, communication.messages.ResetEpochRequest):
+            if reset_epoch_fn is not None:
+                reset_epoch_fn(source_datapipe, request.args)
```

Comment:
Should we assume `args` as a list and do `*request.args`?

Reply:
Done